index.js

```diff
@@ -1,10 +1,11 @@
 const fs = require('fs');
 
-function AssetFingerprint(initializerDirectory, needsFingerprint = true) {
+function AssetFingerprint(initializerDirectory, needsFingerprint = true, fingerprintName) {
   _validateInitializerDirectory();
 
   this.initializerDirectory = initializerDirectory;
   this.needsFingerprint = needsFingerprint;
+  this.fingerprintName = fingerprintName || 'ASSET_FINGERPRINT';
```

Comment:
I'm wondering if it might be beneficial to have some sort of validation/constant format enforcement here to ensure something valid is written to the Ruby file.

Reply:
I could add a check for names that have A-Z and  _ in the middle and throw an error when it does not match.

Reply:
On that note, do we want to lock them to something like this to enforce a convention, raise a meaningful error if not, and add something to the docs? Or leave off the "FINGERPRINT" part but still enforce the rest:
```
/^([A-Z]|_)*_FINGERPRINT$/
```
This would allow for something like "TEST_THIS_FINGERPRINT" as well.